tools/assets/extract/extase/__init__.py

```diff
@@ -20,6 +20,11 @@
 VERBOSE_FILE_TRY_PARSE_DATA = 0
 VERBOSE_REPORT_RESBUF = False
 
+DUMP_REPORTERS_IN_SOURCE = False
+DUMP_XML_IN_SOURCE = False
+
+DONT_MERGE_RESOURCE_BUFFERS_FROM_DIFFERENT_USERS = False
```

Comment:
Changing source code just for a setting doesn't sound great. 
How much are these constants meant to be changed? If not much then I guess it doesn't matter

Reply:
I think they can come in quite handy while doing assets docs passes, but that's the extent of it
I could externalize them to a settings-specific file, like, in [`tools/assets/conf.py`](https://github.com/zeldaret/oot/blob/main/tools/assets/conf.py) ?

Reply:
Hmm, I see. I think it is fine either way then